### ISSUE TITLE
WordStream Maker for NB

### ISSUE THREAD

**Issue by brusboks** [NationalLibraryOfNorway/DHLAB#153]
Hei,

Vil det være mulig å få til en norsk versjon av wordstream maker som kan brukes på NB-corpus?

https://huyen-nguyen.github.io/maker/index.html 

Takk :)

**Comment by Yoonsen** [comment]
Ja, det lar seg gjøre å generere data fra NB som kan gå inn i den typen visualiseringer. 
Eller tenkte du noe utover det?

**Comment by brusboks** [comment]
Beklager at jeg glemte å svare på denne, det var virkelig ikke meningen. Jeg var jo interessert i å følge opp. 

Uansett, jeg følger opp nå. på Hackathon i februar så jeg behovet av å ha en visualiering som kunne si mer enn en ren ordsky gjør. Wordstream utnytter timestamp som metadata for å gi en visuell fremstilling av ord over tid for et korpus slik at det er visuelt mulig å se ordbruk over tid. Imidlertid er jeg litt usikker på om det å bruke wordstream på et norsk korpus vil fungere like bra. Særlig når det kommer til POS og NER for å kategorisere. Her trengs et NPL library for norsk som kan være valgbart gjennom interface i så fall. 

Hvis så - kan det utvikles en egen NB-app som er en norsk versjon av wordstream? 
Eller hva tenker du? :)
